### DISCUSSION TITLE
i need the password to my vps

### DISCUSSION THREAD

**Discussion by 0304cachito** [ovh/manager#4424] [Q&A] [answered]
i need the password to my vps

**Comment by antleblanc** [comment, marked as answer]
Hi @0304cachito!

Could you please refer to this following guide: 
https://docs.ovh.com/gb/en/vps/root-password/

It will helps you to recover your VPS password.

Thank you!

**Reply by wartono-prog** [reply]
my order "Awaiting supporting documents''at status tracing order. can u help me? this status sinje 21 jan 2021.
